source/common/http/codes.cc

```diff
@@ -18,111 +18,132 @@
 namespace Envoy {
 namespace Http {
 
-void CodeStatsImpl::chargeBasicResponseStat(Stats::Scope& scope, const std::string& prefix,
+CodeStatsImpl::CodeStatsImpl(Stats::SymbolTable& symbol_table)
+    : stat_name_pool_(symbol_table), symbol_table_(symbol_table), canary_(makeStatName("canary")),
+      canary_upstream_rq_time_(makeStatName("canary.upstream_rq_time")),
+      external_(makeStatName("external")),
+      external_rq_time_(makeStatName("external.upstream_rq_time")),
+      external_upstream_rq_time_(makeStatName("external.upstream_rq_time")),
+      internal_(makeStatName("internal")),
+      internal_rq_time_(makeStatName("internal.upstream_rq_time")),
+      internal_upstream_rq_time_(makeStatName("internal.upstream_rq_time")),
+      upstream_rq_1xx_(makeStatName("upstream_rq_1xx")),
+      upstream_rq_2xx_(makeStatName("upstream_rq_2xx")),
+      upstream_rq_3xx_(makeStatName("upstream_rq_3xx")),
+      upstream_rq_4xx_(makeStatName("upstream_rq_4xx")),
+      upstream_rq_5xx_(makeStatName("upstream_rq_5xx")),
+      upstream_rq_unknown_(makeStatName("upstream_rq_Unknown")),
```

Comment:
Capital "U" in unknown. Can you make sure we have complete tests for every stat here? I'm worried we might not have full coverage. Also, what's "unknown"? Is this a new stat?

Reply:
There was no coverage for 1xx; added it. I tested all the other initialized strings here by making sure a test failed when I added a stray `x`.